EIPS/EIP-X-add-token-uri-kylanhurt.md

```diff
@@ -0,0 +1,60 @@
+
+## Motivation
+The convenience of representing payment requests by standard URLs has been a significant factor in the cryptocurrency. Extending this convenient mechanism to support embedding of basic token information would help to facilitate the usage of Ethereum-based tokens. In particular, QR-code embedded URLs could be generated by token-generators to help users track the tokens via any wallet software that has the ability to scan QR codes (particularly mobile wallets). For example, rather than users needing to find the contract address and number of decimal places for a given token through an online directory, they could simply scan a QR-code provided by the token generator with this data embedded. The wallet software could then automatically add the tokens to the user's wallet. This would improve the user experience associated with purchasing and holding tokens as a digital asset.
+
+
+## Specification
+
+### Syntax
+Token data URLs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+
+    request                 = "ethereum" ":" [ "token-" ]target_address [ "@" chain_id ] [ "?" parameters ]
+    target_address          = ethereum_address
+    chain_id                = 1*DIGIT
+    ethereum_address        = ( "0x" 40*40HEXDIG ) / ENS_NAME
+    parameters              = parameter *( "&" parameter )
+    parameter               = key "=" value
+    key                     = "symbol" / "decimals" / "name" / "type"
+    value                   = STRING / number (0-18) / STRING / STRING
+
+Where all of the key + value pairs are optional, allowing for maximum flexibility on the application layer.
+
+
+### Semantics
+
+`target_address` is mandatory and denotes either the beneficiary of native token payment (see below) or the contract address with which the user is asked to interact.
+
+`chain_id` is optional and contains the decimal chain ID, such that transactions on various test- and private networks can be requested. If no `chain_id` is present, the client's current network setting remains effective.
+
+If using ENS names instead of hexadecimal addresses, the resolution is up to the payer, at any time between receiving the URL and sending the transaction. Hexadecimal addresses always take precedence over ENS names, i. e. even if there exists a matching ENS name consisting of 40 hexadecimal digits, it should never be resolved. Instead, the hexadecimal address should be used directly.
+
+* `symbol` (optional) is the token symbol (STRING) - Defaults to "SYM"
+* `decimals` (optional) is a whole number between 0 and 18 (inclusive) - Defaults to 18
+* `name` (optional) is a more user-friendly name (STRING) for the token, typically the name of the project that generated the tokens - Defaults to the value of symbol
+* `type` (optional) is the type of token (STRING), eg "ERC20". - Defaults to "ERC20"
```

Comment:
eg should be e.g.
Also wondering if this should be a list - and perhaps in the form like defined in #881